tests/instruments/test_common_base.py

```diff
@@ -114,7 +114,14 @@ def fake():
 
 class ExtendedBase(FakeBase):
     # Keep values unchanged, just derive another instrument, e.g. to add more properties
-    pass
+    fake_ctrl2 = CommonBase.control(
+        "", "%d", "docs",
+        validator=truncated_range,
+        values=(1, 10),
+        dynamic=True,
+    )
+
+    fake_ctrl2.values = (2, 20)
```

Comment:
```suggestion
    fake_ctrl2_values = (2, 20)
```
Shouldn't be like this ?

Reply:
Yes...

I just verified, that all tests pass with this change.

Reply:
Funny that the tests passed before, too... is that something we should investigate?

Reply:
I already thought about it. I used "truncated" instead of "strict".
Now I fixed it and expanded the test.